test/api/artifact_test.js

```diff
@@ -11,23 +11,21 @@ suite('Post artifacts', function() {
   var Bucket        = require('../../queue/bucket');
   var BlobStore     = require('../../queue/blobstore');
   var data          = require('../../queue/data');
-  var helper        = require('./helper')();
-
-  // Create datetime for created and deadline as 3 days later
-  var created = new Date();
-  var deadline = new Date();
-  deadline.setDate(created.getDate() + 3);
+  var base          = require('taskcluster-base');
+  var taskcluster   = require('taskcluster-client');
+  var expect        = require('expect.js');
```

Comment:
`should.js` is nice... but `should(a).be.equal.to(b)` reads wrong. And I don't like `a.should.be.equal.to(b)` because it modifies prototype.

What do you think of `assume` https://www.npmjs.com/package/assume
I'm starting to consider it for new stuff... I think it's a good name, more updated than `expect`...

But also reads nicely without polluting the `Object.prototype` which I dislike on principal, granted `should` probably doesn't give many problems....